pymeasure/instruments/attocube/anc300.py

```diff
@@ -211,12 +212,18 @@ def check_errors(self):
 class ANC300Controller(Instrument):
     """ Attocube ANC300 Piezo stage controller with several axes
 
-    :param host: host address of the instrument
+    :param adapter: The VISA resource name of the controller
+        (e.g. "TCPIP::<address>::<port>::SOCKET") or a created Adapter.
```

Comment:
Is the port configurable? Below in "handle_host", you give a fixed port. If the port is fixed, you can still leave it as "<port>", but could add another line, that the port is 7230.

Reply:
let me know if this is sufficient